docs/pages/docs/guides/custom-admin-ui-navigation.mdx

```diff
@@ -0,0 +1,416 @@
+import { Markdown } from '../../../components/Markdown';
+
+# Custom Admin UI Navigation
+
+In this guide, we'll show you how to create a custom Navigation component to be rendered in the Admin UI.
+
+!> By the end of this guide you should have a custom Navigation component rendering in the Admin UI with a custom route pointing to the KeystoneJS docs.
+![finished custom navigation example](/assets/guides/custom-admin-ui-navigation/final-result.png)
+
+## Adding the custom navigation component to the Admin UI
+
+The first thing you'll need to do in creating your custom Navigation component is to specify an `/admin/config.ts` file in the root of your Keystone project.
+
+Keystone is made aware of custom components it needs to render in the Admin UI via the `/admin/config.ts` file.
+Below is the expected type for the component export:
+
+```ts
+export type AdminConfig = {
+  components?: {
+    Logo?: (props: {}) => ReactElement;
+    Navigation?: (props: NavigationProps) => ReactElement;
+  };
+};
+```
```

Comment:
```suggestion
```

This info isn't really helpful at this stage. We can just show them what to put in file, they can worry about typescript later.